packages/network-controller/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0]
+### Changed
+- Relax dependencies on `@metamask/base-controller` and `@metamask/controller-utils` (use `^` instead of `~`) ([#998](https://github.com/MetaMask/controllers/pull/998))
+- Rename `provider` in `NetworkState` to `providerConfig` ([#995](https://github.com/MetaMask/controllers/pull/995))
```

Comment:
This should be labelled as breaking

Reply:
Oh, we should also mention the change to the controller messenger event. And maybe advise a state migration? Just so it's crystal clear that a migration is needed.

Reply:
Updated in [a24e1cc](https://github.com/MetaMask/controllers/pull/999/commits/a24e1ccf93fe97dd79dcba59200b3a43bdd4bf95).